style.css

```diff
@@ -657,3 +683,726 @@ ul li {
+    padding: 0;
+    gap: 8px;
+    width: 258px;
+    height: 40px;
+    left: 1039px;
+    top: 16px;
+  }
+
+  .mylogo {
+    width: 76px;
+    height: 20px;
+    margin-left: 155px;
+    margin-top: -10px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #6070ff;
+  }
+
+  .navbar-links ul {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 18px;
+    position: absolute;
+    width: 258px;
+    height: 40px;
+    left: 0;
+    top: 0;
+  }
+
+  .main {
+    background-image: url(img/Header\ bg@2x.png);
+    background-size: 100% 100%;
+    background-position: 100% 100%;
+    height: 721.4596557617188px;
+    width: 1397.3985595703125px;
+    left: 34.6015625px;
+    top: 119px;
+    border-radius: 0;
+  }
+
+  main h1 {
+    width: 920px;
+    height: 122px;
+    margin-left: 220px;
+    margin-top: 120px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+  }
+
+  .mytxt {
+    width: 920px;
+    height: 98px;
+    margin-left: 220px;
+    margin-top: 360px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+  }
+
+  .connect {
+    margin-left: 220px;
+  }
+
+  .social {
+    margin-left: 220px;
+  }
+
+  .work {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 142px;
+    gap: 142px;
+    position: relative;
+    width: 1440px;
+    height: 2694px;
+    left: 0;
+    top: 917px;
+  }
+
+  .card1 {
+    position: absolute;
+    height: 496px;
+    width: 1156px;
+    left: 142px;
+    top: 142px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 24px;
+  }
+
+  .cardone .image {
+    display: none;
+  }
+
+  .card1image {
+    background-image: url(img/Snapshoot_Portfoliocard1.png);
+    background-repeat: no-repeat;
+    position: absolute;
+    width: 544px;
+    height: 448px;
+    left: 24px;
+    top: 24px;
+    border-radius: 8px;
+  }
+
+  .card1txt {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 24px;
+    gap: 24px;
+    position: absolute;
+    width: 544px;
+    height: 445px;
+    left: 590px;
+    top: 24px;
+  }
+
+  .cardone h2 {
+    width: 496px;
+    height: 46px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #172b4d;
+  }
+
+  .conpy {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 12px;
+    width: 320px;
+    height: 40px;
+    margin-left: 0;
+  }
+
+  /* .cardone ul {
+    display: flex;
+    align-items: center;
+    padding: 0;
+    gap: 8px;
+    width: 214px;
+    height: 24px;
+    top: 180px;
+  }  */
```

Comment:
- [ ] Kindly, make sure to remove all commented code from your project as suggested by **[HTML CSS best practice]**. This will make your code way cleaner 🧼 👍

Reply:
Done